android/demos/compose/build.gradle

```diff
@@ -1,63 +1,44 @@
-apply plugin: 'com.android.application'
-apply plugin: 'org.jetbrains.kotlin.android'
-apply plugin: "org.jetbrains.kotlin.kapt"
-
-kotlin {
-    jvmToolchain(11)
+plugins {
+    id("ribs.kotlin-android-application-conventions")
+    alias(libs.plugins.kotlinKapt)
 }
 
 android {
     namespace "com.uber.rib.compose"
-    compileSdk deps.build.compileSdkVersion
 
     defaultConfig {
-        minSdk deps.build.minSdkVersion
-        targetSdk deps.build.targetSdkVersion
         applicationId "com.uber.rib.compose"
-        versionCode 1
-        versionName "1.0"
     }
     buildFeatures {
         compose true
     }
     composeOptions {
-        kotlinCompilerExtensionVersion deps.versions.androidx.compose.compiler
-    }
-    compileOptions {
-        sourceCompatibility deps.build.javaVersion
-        targetCompatibility deps.build.javaVersion
-    }
-
-    // No need for lint. This is just a tutorial.
-    lint {
-        abortOnError false
-        quiet true
+        kotlinCompilerExtensionVersion libs.versions.compose.compiler.get()
     }
 }
 
 dependencies {
-    kapt deps.uber.motifCompiler
-    implementation project(":libraries:rib-android")
-    implementation project(":libraries:rib-android-compose")
-    implementation project(":libraries:rib-coroutines")
-    implementation deps.androidx.activityCompose
-    implementation deps.androidx.annotations
-    implementation deps.androidx.appcompat
-    implementation deps.androidx.composeAnimation
-    implementation deps.androidx.composeFoundation
-    implementation deps.androidx.composeMaterial
-    implementation deps.androidx.composeNavigation
-    implementation deps.androidx.composeRuntimeRxJava2
-    implementation deps.androidx.composeUi
-    implementation deps.androidx.composeViewModel
-    implementation deps.androidx.composeUiTooling
-    implementation deps.androidx.savedState
-    implementation deps.external.rxandroid2
-    implementation deps.kotlin.coroutines
-    implementation deps.kotlin.coroutinesAndroid
-    implementation deps.kotlin.coroutinesRx2
-    implementation deps.uber.autodisposeCoroutines
-    implementation deps.uber.motif
+    kapt(libs.motif.compiler)
```

Comment:
Not that we should increase the scope of this diff, but we should move this to KSP motif.

Tracking here https://github.com/uber/RIBs/issues/591